ext/fast_xirr/common.c

```diff
@@ -11,13 +11,13 @@
  *
  * @return The calculated NPV.
  */
-double npv(double rate, CashFlow *cashflows, long long count, time_t min_date) {
+double npv(double rate, CashFlow *cashflows, long long count, int64_t min_date) {
     double npv_value = 0.0;
 
     for (long long i = 0; i < count; i++) {
         // Calculate the number of days from the minimum date to the cash flow date
-        double days = difftime(cashflows[i].date, min_date) / (60 * 60 * 24);
-        
+        double days = (double)(cashflows[i].date - min_date) / (60 * 60 * 24);
```

Comment:
This was tricky to catch. `difftime` uses `time_t` under the hood ([here](https://github.com/lattera/glibc/blob/master/time/difftime.c) the definition), and this causes some cross platform discrepancies.

As the calculation was really easy, I decided to implement it here inline using `int64_t` to assure the same behaviour across different archs and OS